### DISCUSSION TITLE
Support of Musicbrainz database

### DISCUSSION THREAD

**Discussion by hikerus** [enzo1982/freac#421] [Help & Support]
Is there any possibility to use the database of Musicbrainz with freac to get the CD information? The Freedb gateway to Musicbrainz doesn't work any more, so is there an other connection URL?

**Comment by enzo1982** [comment]
No, after the MusicBrainz freedb gateway has been discontinued, it's currently not possible to use MusicBrainz with fre:ac.

This will change, however, as I plan to add dedicated MusicBrainz support to fre:ac later this year.

**Comment by DavidPesticcio** [comment]
As a workaround, I use [Picard](https://picard.musicbrainz.org/) to lookup CD, then use the closest match found in CDDB by fre:ac to rip, then use Picard to populate the remainder of the information afterwards.

HTH

**Comment by Sebrof** [comment]
Hello, 
Have you been able to make any progress with using Musicbrainz with Fre:ac?  I have started using Fre:ac in Haiku, and really hope that there is a way to add track data.
Thanks
